test/src/tests-utils.ts

```diff
@@ -369,8 +370,50 @@ describe('Utils', () => {
 
         it('returns true if Custom Flag Property is not valid', () => {
             expect(isValidCustomFlagProperty(null), 'null').to.equal(false);
-            expect(isValidCustomFlagProperty(function(){}), 'function').to.equal(false);
-            expect(isValidCustomFlagProperty(undefined), 'undefined').to.equal(false);
+
+    describe('#mergeObjects', () => {
+        it('should merge objects', () => {
+            const obj1 = { foo: 'bar' };
+            const obj2 = { fizz: 'buzz' };
+
+            expect(mergeObjects<any>(obj1, obj2)).to.deep.equal({
+                foo: 'bar',
+                fizz: 'buzz',
+            });
+        });
+
+        it('should override values with the same keys', () => {
```

Comment:
Does this make more sense?  I was having a hard time trying to think of something that made sense.

```suggestion
        it('should allow later arguments to override values of previous arguments with the same keys', () => {
```

Reply:
What about `should override values with the same keys in the order they appear`?